src/components/explore/ToplineRow.svelte

```diff
@@ -0,0 +1,109 @@
+<script>
+import { tweened } from 'svelte/motion';
+import { tooltip as tooltipAction } from 'udgl/utils/tooltip';
+import Help from 'udgl/icons/Help.svelte';
+import { formatToBuildID } from '../../utils/formatters';
+
+export let value;
+export let compare;
+export let aggregationLevel;
+export let description;
+
+export let params = { duration: 250 };
+const tw = tweened(+value, params);
+$: $tw = +value;
+
+let parsedLabel = formatToBuildID(new Date($tw));
+$: parsedLabel = formatToBuildID(new Date($tw));
+
+function delta(a, b, al) {
+  if (al === 'build_id') {
+    if (!b) return undefined;
+    const hours = (+(b) - +(a)) / 1000 / 60 / 60;
+    if (hours < 24 && hours > -24) return `${~~hours} hour${~~hours === 1 ? '' : 's'}`;
```

Comment:
Four eslint errors on this line:

```
  23:46  error  Unexpected use of '~'  no-bitwise
  23:47  error  Unexpected use of '~'  no-bitwise
  23:61  error  Unexpected use of '~'  no-bitwise
  23:62  error  Unexpected use of '~'  no-bitwise
```

Reply:
Maybe we should use `Math.floor` here. Could just be me, but I hadn't seen this before and it did confuse me. `Math.floor` seems more readable.